components/PopupMenu/PopupMenu.js

```diff
@@ -19,11 +19,18 @@ export default function PopupMenu({
         edit
       </StyledMenuButton>
       {pathname === "/[category]" && (
-        <Link href={`/declension/${id}`}>
-          <StyledMenuButton>
-            {prop.query1?.declension ? "declension" : "+ declension"}
-          </StyledMenuButton>
-        </Link>
+        <>
+          <Link href={`/declension/${id}`}>
+            <StyledMenuButton>
+              {prop.query1?.declension ? "declension" : "+ declension"}
+            </StyledMenuButton>
+          </Link>
+          <Link href={`/notes/${id}`}>
+            <StyledMenuButton>
+              {prop.notes ? "notes" : "+ notes"}
```

Comment:
i love the little extra UX with the "+" before the word❤️

Reply:
I totally agree :)